terraform/scripts/update_geoip.sh

```diff
@@ -71,7 +71,7 @@ if [ "$geoip_remote_md5" != "$geoip_local_md5" ]; then
   exit
 fi
 
-scp "$geoip_local_file" "$target_name":
+scp -p "$geoip_local_file" "$target_name":
```

Comment:
Note that `scp` does not offer a long flag for this option.

Reply:
Yes, unfortunately. Would you like me to add a comment mentioning what the flag does?

Reply:
That wasn't my intent, but now that you mention it it isn't a bad idea.